bedrock/firefox/templates/firefox/android/sms-send.html

```diff
@@ -21,7 +21,7 @@ <h1 class="large">{{_('Get Firefox for Android')}}</h1>
       Send yourself a text message <br>to download it from Google Play.
     {% endtrans %}
     </h2>
-    <p class="note">{{ _('SMS service available in the U.S. only') }}</p>
+    <p class="note">{{ _('SMS service available in the U.S. only. SMS &amp; data rates may apply. The intended recipient of the SMS must have consented. <a href="%s">Learn more</a>')|format(url('privacy.notices.websites') + '#campaigns') }}</p>
```

Comment:
I guess the escape issue is site-wide. See https://www.mozilla.org/en-US/firefox/37.0/whatsnew/